skopt/callbacks.py

```diff
@@ -328,5 +328,5 @@ def load(self):
         if os.path.exists(self.checkpoint_path):
             result = load(self.checkpoint_path)
             return {'x0': result.x_iters, 'y0': result.func_vals,
-                    'base_estimator': result.models[-1]}
+                    'base_estimator': result.models[-1] if result.models else None}
```

Comment:
Not sure if this isn't a different bug: https://github.com/scikit-optimize/scikit-optimize/issues/1003

Reply:
From what I have seen `results.model` will not be defined until your evaluation has finished the warm up stage, computation of `n_initial_points` picked by a process (typically random).